### DISCUSSION TITLE
Your App is great, but i got some issues

### DISCUSSION THREAD

**Discussion by cahyo47wibowo** [Shashank02051997/QuizzieThunder-Flutter#2] [General]
when i try to register new account, i don't get the OTP response ( i tried it with dummy phone number https://receive-smss.com) because im foreigner doesn't have india phone number +91
maybe can you give alternative dummy phone number, so i can register it or may be can you give me some account so i can use it.
It will help me a lot to learn flutter, Thank you very much for sharing
![Uploading Screenshot 2023-12-16 at 01.04.07.png…]()

**Comment by Shashank02051997** [comment]
Thank you for using our app and expressing your interest in learning Flutter! We appreciate your support. To help you get started, here are some credentials you can use to log in to the mobile app:

Phone number: 6785432111
Password: sandeep

Feel free to explore the app and enjoy your learning journey with Flutter. If you have any further questions or need assistance, don't hesitate to reach out. Happy coding!

**Comment by cahyo47wibowo** [comment]
That's really great your app can run on IOS and Android Smoothly, i hope I
can learn to code from your project.
Thanks a lot for giving me some acount to testing.
I hope you're always happy. I hope you're always healthy. I hope you're
always successful

Kind regards
Cahyo
